book/07-git-tools/sections/submodules.asc

```diff
@@ -437,6 +437,19 @@ This will make Git run `git submodule update` right after the pull, putting the
 Moreover, if you want to make Git always pull with `--recurse-submodules`, you can set the configuration option `submodule.recurse` to true (this works for `git pull` since Git 2.15).
 This option will make Git use the `--recurse-submodules` flag for all commands that support it (except `clone`).
 
+There is a special situation that can happen when pulling superproject updates : maybe the upstream repository changes the URL of the submodule in the `.gitmodules` file in one of the commits you pull.
```

Comment:
```suggestion
There is a special situation that can happen when pulling superproject updates: it could be that the upstream repository has changed the URL of the submodule in the `.gitmodules` file in one of the commits you pull.
```